### PR TITLE
Add component return to mapper

### PR DESCRIPTION
**Pull request by rvegas**

### CODE REVIEW

**Comment by orestes** [comment]
The build passed but for some reason it did not ping GitHub back, weird...

File: sdk/mappings.js [hunk 1]
```diff
@@ -81,6 +81,7 @@ const Mappings = {
   'required': 'r',
   'response': 'R',
   'result': 'r',
+  'response_result': 'rv',
```
Comment (orestes) [inline sdk/mappings.js:84]: This should be `response_return` I think.

File: sdk/mapper.js [hunk 2]
```diff
@@ -65,8 +66,13 @@ class Mapper {
         }
       }
     };
+    if (responseApi.hasReturn()) {
+      payload[m.command_reply][m.result][m.response_result] = responseApi.getReturnType();
```
Comment (orestes) [inline sdk/mapper.js:70]: `m.response_result` should be `response_return` I think.

**Review by orestes — APPROVED** [review]
Minor typo, not important at the moment

File: sdk/mappings.js (hunk 1 above)
Comment (rvegas) [reply sdk/mappings.js:84]: indeed. will make another pr later.

File: sdk/mapper.js (hunk 2 above)
Comment (rvegas) [reply sdk/mapper.js:70]: indeed. will make another pr later.

**Pull request merged**